src/pyubx2/ubxreader.py

```diff
@@ -84,6 +84,8 @@ def __init__(self, datastream, **kwargs):
         self._labelmsm = int(kwargs.get("labelmsm", True))
         self._msgmode = int(kwargs.get("msgmode", GET))
 
+        self._parsing = kwargs.get("parsing", True)

```

Comment:
this is fine, but need to update docstring at the top of the module:

:param bool parsing: a boolean flag to determine if the stream needs to be parsed

(you can remove this docstring entries from the parse_* methods as it no longer applies)

Reply:
Added